pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
+	if err != nil {
+		return restoreWaitUnknown
+	} else if strings.Contains(string(bytes), "[-w secs]") {
```

Comment:
@danwinship RHEL7 currently ships iptables 1.4.21 which we first built for RHEL on Jan 9 2014.  Since then we've backported about 6 patches from git.  I seriously doubt that RHEL will just rebase iptables to 1.6.2 whenever that deigns to come out.

What about a two-stage solution: first check version, then if that fails fall back to --help parsing?